### PR TITLE
TD: fix header uniformity

### PR DESCRIPTION
**Pull request by luzpaz**
This PR fixes header uniformity across all TechDraw WB.

### CODE REVIEW

File: src/Mod/TechDraw/App/AppTechDrawPy.cpp [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /***************************************************************************
- *   Copyright (c) Jürgen Riegel          (juergen.riegel@web.de) 2002     *
- *   Copyright (c) WandererFan            (wandererfan@gmail.com) 2016     *
-*                                                                         *
+ *   Copyright (c) 2002 Jürgen Riegel <juergen.riegel@web.de>              *
+ *   Copyright (c) 2016 WandererFan <wandererfan@gmail.com>                *
+ *                                                                         *
```
Comment (vocx-fc) [inline src/Mod/TechDraw/App/AppTechDrawPy.cpp]: Personally, I don't think you need to repeat the word Copyright for each author. It's obvious after the first one that all of them have that copyright. What's important is the year, to know when an author did a significant change.

Comment (luzpaz) [reply src/Mod/TechDraw/App/AppTechDrawPy.cpp]: I don't know.

Comment (luzpaz) [reply src/Mod/TechDraw/App/AppTechDrawPy.cpp]: I think we should just move forward with this PR as is. We can discuss the cosmetic change later.

**Comment by WandererFan** [comment]
Don't really see the point of this one, but it doesn't hurt anything,  so merged.

**Pull request merged**